### DISCUSSION TITLE
local  selt hosted AI

### DISCUSSION THREAD

**Discussion by tekjunkie6502** [excalidraw/excalidraw#8239] [General]
I am wondering if there are plans to make a local self-host AI connection options?
I am just not a fan of using at this moment with all big tech.
I have my own local ollama and Stable Diffusion locally.

JP

**Comment by orkutmuratyilmaz** [comment]
Any updates on this?

**Reply by Mrazator** [reply]
Hey, there isn't any progress on this currently, linking related discussion: https://github.com/excalidraw/excalidraw/issues/7389
